doc/source/release-notes/Percona-Server-5.6.26-74.0.rst

```diff
@@ -0,0 +1,72 @@
+.. rn:: 5.6.26-74.0
+
+==============================
+ |Percona Server| 5.6.26-74.0 
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.26-74.0 on September 15th, 2015 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.26-74.0/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.26 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-26.html>`_, including all the bug fixes in it, |Percona Server| 5.6.26-74.0 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.26-74.0 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.26-74.0>`_.
+
+New Features
+============
+
+ |TokuDB| storage engine `source <https://github.com/percona/tokudb-engine.git>`_ has been merged into the |Percona Server| code. |TokuDB| storage engine version has been replaced by the |Percona Server| version. 
```

Comment:
I think all merged repos should be mentioned, especially since their docs are being merged in as well.

TokuDB SE used to maintain its own version (i.e. "tokudb-7.5.8") where as now it just inherits the server version similar to XtraDB.